src/Schema/SimulationPlotSchema_0.0.1.json

```diff
@@ -4,59 +4,164 @@
+      "$ref": "#/definitions/scenarioSchema"
+    }
+  ,
+    "^(?!version)[\\w\\d]{1,30}$" : {
+      "$ref": "#/definitions/operationSchema"
+    }
+  },
+  "properties": {
+    "version":{
+      "type": "string",
+      "enum": [
+        "0.0.1"
+      ]
+    }
+  },
+  "required": ["version"],
+
+  "definitions": {
+    "scenarioSchema" : {
+      "title": "scenarioSchema.json",
+      "type": "array",
+      "definitions": {},
+      "items": {
+        "oneOf": [
+          {
+            "type": "string"
+          },
+          {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+              "eps": {
+                "description": "Events per second being emitted",
+                "type": "integer"
+              },
+              "priority" : {
+                "type": "integer",
+                "description": "This determines whether it should be prioritized in case the resources to emit events is insufficient "
+              },
+              "payload": {
+                "type": "object",
+                "additionalProperties": true,
+                "description": "The payload that is going to be sent is currently specifically defined."
+              },
+              "parameters": {
+                "type": "object",
+                "description": "Specify the parameter instance variables of the channel, if any are present.",
+                "additionalProperties": false,
+                "patternProperties": {
+                  "^[\\w\\d]{1,28}$" : {
+                    "oneOf": [
+                      {
+                        "type": "string"
+                      },
+                      {
+                        "type": "integer"
+                      },
+                      {
+                        "type": "object",
+                        "additionalProperties": false,
+                        "required": ["regex"],
+                        "properties": {
+                          "regex": {
+                            "type": "string"
+                          }
+                        }
+                      },
+                      {
+                        "type": "object",
+                        "additionalProperties": false,
+                        "required": ["range"],
+                        "properties": {
+                          "range": {
+                            "type": "object",
+                            "properties": {
+                              "min": {
+                                "type": "integer"
+                              },
+                              "max": {
+                                "type": "integer"
+                              }
+                            }
+                          }
+                        }
+                      }
+                    ]
+                  }
+                }
+              }
+            }
+          }
+        ]
       }
```

Comment:
IMO each object here, should be split into their own definition, easier to read 🙂